service/tablet_allocator.cc

```diff
@@ -3104,53 +3171,35 @@ class load_balancer {
             if (node.drained) {
                 continue;
             }
-            if (!node.capacity) {
+            if (!node.dusage) {
                 lblogger.info("Cannot balance because capacity of node {} (or more) is unknown", host);
                 co_return plan;
             }
         }
```

Comment:
You can now fold this loop into the later one, which checks for data validity in load_stats